content/utilities/map-export/plugin.js

```diff
@@ -68,6 +61,37 @@ class Plugin {
       this.status.style.color = 'red';
     }
   }
+  
+  onImport = async () => {
+    let input;
+    try {
+      input = await window.navigator.clipboard.readText();
+      this.processMap(input);
+    } catch (err) {
+      console.error(err);
+      this.status.innerText = 'Unable to import map. Did you allow clipboard access?';
+      this.status.style.color = 'red';
+      return;
+    }
```

Comment:
```suggestion
    let input;
    try {
      input = await window.navigator.clipboard.readText();
    } catch (err) {
      console.error(err);
      this.status.innerText = 'Unable to import map. Did you allow clipboard access?';
      this.status.style.color = 'red';
      return;
    }
    this.processMap(input);
```

I moved `this.processMap` out of the try/catch because the error message is very specific to no clipboard access

Reply:
I'm going to commit these so I can test